third_party/move/move-vm/test-utils/src/gas_schedule.rs

```diff
@@ -346,6 +346,24 @@ impl GasMeter for GasStatus {
         )
     }
 
+    fn charge_pack_closure(
```

Comment:
Honestly we should've killed `GasStatus` already -- it's the legacy gas meter with no practical use.